tests/functional/run-action-with-multiple-execute-steps/expected-stdout

```diff
@@ -33,5 +33,4 @@ Found 2 models, 2 tests, 0 snapshots, 0 analyses, %s macros, 0 operations, 0 see
 Concurrency: 1 threads (target='kbc_prod')
 1 of 2 START freshness of in.c-test-bucket.test %s [RUN]
 2 of 2 START freshness of out.c-my-dbt-transformation-transformation.fct_model %s [RUN]
-2 of 2 PASS freshness of out.c-my-dbt-transformation-transformation.fct_model %s [PASS in %ss]
```

Comment:
Tohle nevim :D Uz jsem to jednou odebral, pak zas pridal a ted zas odebral :D

Reply:
dpc, to fakt nechapem :D raz tak raz tak:
https://github.com/keboola/dbt-transformation/actions/runs/3066979052/jobs/4952794221

Reply:
To podle mě zálěží jestli ten `_timestamp` u tabulky je menší než 24 hodiny nebo ne.

Reply:
jo takhle, to by davalo zmysel. Dam tam nejaky placeholder